packages/launcher/lib/detect.ts

```diff
@@ -73,12 +73,13 @@ function checkOneBrowser(browser: Browser) {
 }
 
 /** returns list of detected browsers */
-function detectBrowsers(): Bluebird<Browser[]> {
+function detectBrowsers(goalBrowsers?: Browser[]): Bluebird<Browser[]> {
   // we can detect same browser under different aliases
-  // tell them apart by the full version property
+  // tell them apart by the name and the version property
   // @ts-ignore
-  const removeDuplicates = uniqBy(prop('version'))
-  return Bluebird.mapSeries(browsers, checkOneBrowser)
+  if (!goalBrowsers) goalBrowsers = browsers
```

Comment:
hmm, we don't require `{ }` around code blocks? like it should be
```
if (!goalBrowsers) {
  goalBrowsers = browsers
}
```

Reply:
Looks like the linter doesn't run on specfiles, I'll see about changing that